build.gradle

```diff
@@ -125,6 +125,7 @@ dependencies {
   compile group: 'org.twitter4j', name: 'twitter4j-core', version: '4.0.2'
   compile group: 'org.twitter4j', name: 'twitter4j-stream', version: '4.0.2'
   compile group: 'xerces', name: 'xerces', version: '2.4.0'
+  compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
@sudheesh001 @yukiisbored ok, got it, I will keep these points in mind.

I will try to go through the boilerpipe library and try to implement it's logic to be able to merge some parts of it with genericscraper . 

For time being I shall close this PR